src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileRouteSpec.scala

```diff
@@ -216,8 +216,8 @@ class FileRouteSpec extends TestContainerUtils with Matchers with TestRequest {
   }
 
   private def getReferencesMockServer(additionalRefs: Int = 0): WireMockServer = {
-    val wiremockServer = new WireMockServer(8080)
-    WireMock.configureFor("localhost", 8080)
+    val wiremockServer = new WireMockServer(2234)
```

Comment:
As above - suggest changing to a different alternate http port (e.g. 8008) for consistency